classes/class-p4-settings.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+if ( ! class_exists( 'P4_Settings' ) ) {
+	/**
+	 * Class P4_Settings
+	 */
+	class P4_Settings {
+
+		/**
+		 * Option key, and option page slug
+		 * @var string
+		 */
+		private $key = 'planet4_options';
+
+		/**
+		 * Array of metaboxes/fields
+		 * @var array
+		 */
+		protected $option_metabox = array();
+
+		/**
+		 * Options Page title
+		 * @var string
+		 */
+		protected $title = '';
+
+		/**
+		 * Options Page hook
+		 * @var string
+		 */
+		protected $options_page = '';
+
+		/**
+		 * Constructor
+		 */
+		public function __construct() {
+
+			// Set our title
+			$this->title   = __( 'Planet4', 'planet4-master-theme' );
+			$pages         = get_pages();
+
+			$pages_array[] = __( 'Select Page' ,'planet4-master-theme' ) ;
+			foreach ( $pages as $all_pages ) {
+				$pages_array[$all_pages->ID] = __( $all_pages->post_title ,'planet4-master-theme' ) ;
+			}
+
+			$categories         = get_categories();
+
+			$categories_array[] = __( 'Select Category' ,'planet4-master-theme' ) ;
+			foreach ( $categories as $category ) {
+				$categories_array[$category->term_id] = __( $category->cat_name ,'planet4-master-theme' );
+			}
+
+			$this->fields = apply_filters( 'planet4_options', array(
+					array(
+						'name'    => __( 'Select act page', 'planet4-master-theme' ),
+						'id'      => 'select_act_page',
+						'type'    => 'select',
+						'options' => $pages_array
+					),
+
+					array(
+						'name'    => __( 'Select explore page', 'planet4-master-theme' ),
+						'id'      => 'select_explore_page',
+						'type'    => 'select',
+						'options' => $pages_array
+					),
+
+					array(
+						'name'    => __( 'Select category', 'planet4-master-theme' ),
+						'id'      => 'select_category',
+						'type'    => 'select',
+						'options' => $categories_array
+					)
```

Comment:
@sagarsdeshmukh Please, use short arrays [] and add a comma even in the last key => value pair.

Reply:
I will update the array syntax with short array [].